### PR TITLE
RND203-312: Add USB device table

### PR DESCRIPTION
**Pull request by Rafal-Litwinski-Mobica**

### CODE REVIEW

File: sys/dev/ath/if_ath_usb_devlist.h [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#define	USB_VENDOR_DLINK2					0x07d1		/* D-Link */
+#define	USB_VENDOR_LITEON					0x04ca		/* Lite-On Technology */
+#define	USB_VENDOR_NETGEAR					0x0846		/* BayNETGEAR */
+#define	USB_VENDOR_VIA						0x040d		/* VIA */
+#define	USB_VENDOR_ACCTON					0x083a		/* Accton Technology */
+#define	USB_VENDOR_ACTIONTEC				0x1668		/* Actiontec Electronics */
+#define	USB_VENDOR_PANASONIC				0x04da		/* Panasonic (Matsushita) */
+#define	USB_VENDOR_MELCO					0x0411		/* Melco */
+
+#define	USB_PRODUCT_ATHEROS2_AR9271_1		0x1006		/* AR9271 */
```
Comment (Mikolaj-Filar-Mobica) [inline sys/dev/ath/if_ath_usb_devlist.h]: Could we align all columns?

File: sys/dev/ath/if_ath_usb_devlist.h [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	{ USB_VP(USB_VENDOR_NETGEAR, USB_PRODUCT_NETGEAR_WNA1100) },
+	{ USB_VP(USB_VENDOR_VIA, USB_PRODUCT_VIA_AR9271) },
+	{ USB_VPI(USB_VENDOR_ACCTON, USB_PRODUCT_ACCTON_AR9280, ATHN_USB_FLAG_AR7010) },
+	{ USB_VPI(USB_VENDOR_ACTIONTEC, USB_PRODUCT_ACTIONTEC_AR9287, ATHN_USB_FLAG_AR7010) },
+	{ USB_VPI(USB_VENDOR_NETGEAR, USB_PRODUCT_NETGEAR_WNDA3200, ATHN_USB_FLAG_AR7010) },
+	{ USB_VPI(USB_VENDOR_PANASONIC, USB_PRODUCT_PANASONIC_N5HBZ0000055, ATHN_USB_FLAG_AR7010) },
+	{ USB_VPI(USB_VENDOR_MELCO, USB_PRODUCT_MELCO_UWABR100, ATHN_USB_FLAG_AR7010) },
+	{ USB_VPI(USB_VENDOR_ATHEROS2, USB_PRODUCT_ATHEROS2_AR9280, ATHN_USB_FLAG_AR7010) },
+	{ USB_VPI(USB_VENDOR_ATHEROS2, USB_PRODUCT_ATHEROS2_AR9287, ATHN_USB_FLAG_AR7010) }
+};
```
Comment (Mikolaj-Filar-Mobica) [inline sys/dev/ath/if_ath_usb_devlist.h]: I think a blank line at the end of the file is recommended.

File: sys/dev/ath/if_ath_usb_devlist.h (hunk 1 above)
Comment (Rafal-Litwinski-Mobica) [reply sys/dev/ath/if_ath_usb_devlist.h]: All columns were align with tab, in vscode looks good. I will use spaces instead.

File: sys/dev/ath/if_ath_usb_devlist.h (hunk 2 above)
Comment (Rafal-Litwinski-Mobica) [reply sys/dev/ath/if_ath_usb_devlist.h]: ok

**Review by Mikolaj-Filar-Mobica — APPROVED** [review]

**Review by nzulinski-mobica — APPROVED** [review]

**Pull request merged**